tools/tts/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ghcr.io/coqui-ai/tts-cpu
```

Comment:
needs a hash/permanent version specification.

if this bubbles up to include ubuntu, which is banned on /tg/ servers, then this docker container should be manually built to not include ubuntu.